schema.sql

```diff
@@ -10,4 +10,29 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals 
-	ADD species TEXT;
\ No newline at end of file
+	ADD species TEXT;
+
+CREATE TABLE owners (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    full_name TEXT,
+    age INT,
+    primary key(id)
+);
+
+CREATE TABLE species (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name TEXT,
```

Comment:
- [ ] Great job so far. As per project requirement, the data type to use here is a string. `varchar` is most preferable string for storing names. `TEXT` will occupy more space of which it is for storing just name.
**Example**
` name varchar(255),`